ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -491,7 +491,9 @@ metadata:
   name: loki-promtail
   namespace: loki
 EOF
-cat >> "${SHARED_DIR}/manifest_metrics.yml" << EOF
+if [ -n "${LOKI_USE_SERVICEMONITOR:-}" ]; then
```

Comment:
nit: can we use something more explicit than a non-empty value to mark this?  Using `== true` here would allow for `LOKI_USE_SERVICEMONITOR: "false"` in the jobs that didn't need this, and we could call out `false` as the preferred phrasing for that in `documentation` with something like:

> Monitor promtail via ServiceMonitor. Set to 'false' for OCP 4.5 and earlier, where ServiceMonitor CRDs are installed after bootstrap completion.

Although since you already doc "Must be unset" for those in your current tip, I'm ok if you want to leave this as it stands without moving to true/false.

/lgtm
/hold

Feel free to pull the hold if you want to leave this as you have it, or ping me if you update and need a fresh LGTM.

Reply:
>Although since you already doc "Must be unset" for those in your current tip, I'm ok if you want to leave this as it stands without moving to true/false.

Yeah, lets leave it at that and rework later if needed